clients/buckets/bucket_test.go

```diff
@@ -1220,7 +1223,7 @@ func TestUpdate(t *testing.T) {
 
 		client := buckets.NewClient(
 			rest.NewClient(server.URL(), server.Client()),
-			buckets.WithRetrySettings(5, 0, time.Minute))
+			buckets.WithRetrySettings(bucketDurationBetweenTries, bucketMaxWaitDuration))
```

Comment:
Missing comment

Reply:
I have fixed this by renaming the constant and removing the comments